lib/_debugger.js

```diff
@@ -27,6 +27,7 @@ exports.start = function(argv, stdin, stdout) {
   stdout = stdout || process.stdout;
 
   const args = [`--debug-brk=${exports.port}`].concat(argv);
+  console.error('`node debug` is deprecated, use `node inspect` instead.\n');
```

Comment:
If possible, this should be updated to use `process.emitWarning()` and `DeprecationWarning` instead.